Diagram.svg

```diff
@@ -0,0 +1,271 @@
+     id="text3826"
+     font-weight="400"
+     font-style="normal"
+     font-size="0.80"
+     y="17.775238"
+     x="36.387032"
+     transform="scale(0.88983265,1.1238068)">Resource</text>
+  <rect
+     style="fill:#0c29d0;stroke:#ffffff;stroke-width:0.08408554"
+     id="rect3828"
+     rx="0.000"
+     height="2.4096453"
+     width="5.85707"
+     y="17.311153"
+     x="21.246346" />
+  <text
+     style="font-style:normal;font-weight:400;font-size:0.67268419px;font-family:sans;text-anchor:start;fill:#ffffff;stroke-width:0.84085524"
+     id="text3830"
+     font-weight="400"
+     font-style="normal"
+     font-size="0.80"
+     y="16.640085"
+     x="24.255177"
+     transform="scale(0.88983265,1.1238068)">ListResource</text>
+  <rect
+     style="fill:#0c29d0;stroke:#ffffff;stroke-width:0.08003781"
+     id="rect3832"
+     rx="0.000"
+     height="5.0698996"
+     width="10.74835"
+     y="17.114977"
+     x="33.981194" />
+  <text
+     style="font-style:normal;font-weight:400;font-size:0.75525898px;font-family:sans;text-anchor:middle;fill:#ffffff;stroke-width:0.94407374"
+     id="text3834"
+     font-weight="400"
+     font-style="normal"
+     font-size="0.80"
+     y="19.913082"
+     x="39.373829"
+     transform="scale(0.99906327,1.0009376)">Headers</text>
+  <rect
+     x="17.387392"
+     y="16.891439"
+     width="2.8558433"
+     height="5.7281218"
+     rx="0.000"
+     id="rect131"
+     style="fill:#0c29d0;stroke:#ffffff;stroke-width:0.07081106" />
+  <text
+     style="font-style:normal;font-weight:400;font-size:0.67268413px;font-family:sans;text-anchor:start;fill:#ffffff;stroke-width:0.84085518"
+     id="text3830-5"
+     font-weight="400"
+     font-style="normal"
+     font-size="0.80"
+     y="17.829868"
+     x="20.397861"
+     transform="scale(0.88983266,1.1238068)">Verb</text>
+  <rect
+     x="45.852516"
+     y="16.867022"
+     width="2.8558433"
+     height="5.7281218"
+     rx="0.000"
+     id="rect131-7"
+     style="fill:#0c29d0;stroke:#ffffff;stroke-width:0.07081106" />
+  <text
+     style="font-style:normal;font-weight:400;font-size:0.67268413px;font-family:sans;text-anchor:start;fill:#ffffff;stroke-width:0.84085518"
+     id="text3830-5-4"
+     font-weight="400"
+     font-style="normal"
+     font-size="0.80"
+     y="17.802223"
+     x="52.055748"
+     transform="scale(0.88983267,1.1238068)">Status</text>
+  <rect
+     style="fill:#0c29d0;stroke:#ffffff;stroke-width:0.0786015"
+     id="rect3828-3"
+     rx="0.000"
+     height="1.0618817"
+     width="11.613857"
+     y="21.0655"
+     x="21.114571" />
+  <text
+     style="font-style:normal;font-weight:400;font-size:0.67268419px;font-family:sans;text-anchor:start;fill:#ffffff;stroke-width:0.84085524"
+     id="text3830-0"
+     font-weight="400"
+     font-style="normal"
+     font-size="0.80"
+     y="19.467997"
+     x="29.286266"
+     transform="scale(0.88983266,1.1238068)">Fields</text>
+</svg>
```

Comment:
This svg should go to the package?
The project doesn't have a MANIFEST.in

Reply:
You're right. I'm testing it on test.pypi to see how it renders.
I think we still have some issue wiith rendering this docs that I'm going to address.

Reply:
I just saw that PyPI can't render SVG files directly from github. So I added a link to a wiki page [here](https://github.com/olist/pactum/wiki/Architecture-Diagram).